scalafmt-tests/src/test/scala/org/scalafmt/util/DiffTest.scala

```diff
@@ -1,17 +1,27 @@
 package org.scalafmt.util
 
-import org.scalactic.source.Position
+import java.io.File
+import java.nio.file.Paths
+
+import munit.Location
 import org.scalafmt.config.ScalafmtConfig
 
 case class DiffTest(
     name: String,
     filename: String,
-    loc: Position,
+    loc: Location,
     original: String,
     expected: String,
     skip: Boolean,
     only: Boolean,
     style: ScalafmtConfig
 ) {
-  val fullName = s"${loc.fileName}:${loc.lineNumber}: $name"
+  val file = DiffTest.testDir.relativize(Paths.get(loc.path)).toString()
```

Comment:
This is needed to have test names like `vertical-multiline/beforeImplicitKW.stat`